### DISCUSSION TITLE
part-ii/2016-85/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#295] [Question comments]
Part II 2016 Logic and Set Theory: Paper 1, Section II, F 

https://questions.tripos.org/part-ii/2016-85/

**Comment by ArthurConmy** [comment]
<details>
<summary>a)</summary>
True; transfinite induction.
</details>

<details>
<summary>b)</summary>
False consider something in Cantor Normal Form (which is unique) not in this form, e.g $omega^3 + \omega^2 + \omega$
</details>

<details>
<summary>c)</summary>
True because Cantor Normal Form and distributivity of multiplication, and than $\omega \omega^\beta = \omega^\beta$ if and only if $\beta \ge \omega$.
</details>

<details>
<summary>d)</summary>
False, consider $\omega_0$.
</details>

<details>
<summary>e)</summary>
True since all elements of an empty set of ordinals are indeed uncountable: $\alpha^\omega \ge \alpha^2 > \alpha$.
</details>
